packages/react/src/jsx.tsx

```diff
@@ -13,7 +13,14 @@ import {
 
 type ServerComponent = ComponentModelState<ServerEventModelState>
 
-export const jsxs = jsx
+// @ts-ignore
+export const jsxs = (...args) => {
+  // @ts-ignore
+  const d = jsx(...args)
+  // @ts-ignore
+  d.static = true
```

Comment:
We need to somehow annotate on the ComponentDataState that its children were created statically and as such, do not require keys to be present. 

Question: How should we transfer that information over the wire?